tests/output-files/check-output.sh

```diff
@@ -48,6 +48,37 @@ then
     echo "Error: demangled file singlefile.out.demangled.c did not contain expected demangled struct and function name."
```

Comment:
Can you simplify this test by creating a list of patterns + what is testing? Then iterating over the list and printing the error if it couldn't find the pattern?

I think it will make this less repetitive and easy to debug / maintain.

Reply:
Doing this in bash is a pain. Would it make sense to have a Python script for this? I heard that we want to reduce our dependencies on Python.

Reply:
BTW, we want to reduce our runtime dependencies on Python. IMO, for test purpose, I would prefer python over bash.

Reply:
BTW, it shouldn't be that painful to do this in bash either. That's what we do in the regression script.

https://github.com/model-checking/kani/blob/e28256140f0057f9a67cc46e546ebfeae77eacf0/scripts/kani-regression.sh#L44-L65